mappings/net/minecraft/util/profiler/ReadableProfiler.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS aes net/minecraft/util/profiler/ReadableProfiler
```

Comment:
This is not only readable; its current section string can be modified in the methods.

Reply:
It's readable on top of being writeable. The regular profiler can only be used to push/pop section names but get the profiler results.